### ISSUE TITLE
Installation in Colab notebook

### ISSUE THREAD

**Issue by johnnytam100** [a-r-j/graphein#105]
**Describe the bug**
I did the following for installation:
```
!pip install graphein==1.0.11
!pip install folium==0.2.1
```
And I tried to run:

```
from graphein.protein.config import ProteinGraphConfig
from graphein.protein.graphs import construct_graph
config = ProteinGraphConfig()
g = construct_graph(config=config, pdb_code="3eiy")
```

gave

```
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
[<ipython-input-14-3aa322475106>](https://localhost:8080/#) in <module>()
      2 from graphein.utils.pymol import MolViewer
      3 pymol = MolViewer()
----> 4 pymol.fetch('3eiy')
      5 pymol.display()

3 frames
[/usr/lib/python3.7/subprocess.py](https://localhost:8080/#) in _execute_child(self, args, executable, preexec_fn, close_fds, pass_fds, cwd, env, startupinfo, creationflags, shell, p2cread, p2cwrite, c2pread, c2pwrite, errread, errwrite, restore_signals, start_new_session)
   1549                         if errno_num == errno.ENOENT:
   1550                             err_msg += ': ' + repr(err_filename)
-> 1551                     raise child_exception_type(errno_num, err_msg, err_filename)
   1552                 raise child_exception_type(err_msg)
   1553 

FileNotFoundError: [Errno 2] No such file or directory: 'pymol': 'pymol'

```

I also tried to run:

```
# NBVAL_SKIP
from graphein.utils.pymol import MolViewer
pymol = MolViewer()
pymol.fetch('3eiy')
pymol.display()
```

gave:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-15-1d85efe5422a>](https://localhost:8080/#) in <module>()
----> 1 from graphein.protein.config import ProteinGraphConfig
      2 from graphein.protein.graphs import construct_graph
      3 
      4 config = ProteinGraphConfig()
      5 g = construct_graph(config=config, pdb_code="3eiy")

9 frames
[/usr/local/lib/python3.7/dist-packages/distributed/config.py](https://localhost:8080/#) in <module>()
     18 
     19 with open(fn) as f:
---> 20     defaults = yaml.load(f)
     21 
     22 dask.config.update_defaults(defaults)

TypeError: load() missing 1 required positional argument: 'Loader'
```

**Screenshots**
![image](https://user-images.githubusercontent.com/51283097/154647873-1281d280-69d5-4656-8309-f16dd709c2a6.png)

![image](https://user-images.githubusercontent.com/51283097/154647921-8ef7fcb2-5f76-49ce-8144-6f6b5c2a7e12.png)

May I know how to fix it? Thanks!

**Comment by johnnytam100** [comment]
That worked for the cells! Thanks! Unfortunately, another cell:

```
from graphein.protein.visualisation import plot_protein_structure_graph

p = plot_protein_structure_graph(G=g, angle=0, colour_edges_by="kind", colour_nodes_by="element_symbol", label_node_ids=False, node_size_min=2, node_alpha=0.85, node_colour_map=plt.cm.tab10)
plt.suptitle("Atom-level graph. Nodes coloured by their Element")
```

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-7-c327719f0d21>](https://localhost:8080/#) in <module>()
      1 from graphein.protein.visualisation import plot_protein_structure_graph
      2 
----> 3 p = plot_protein_structure_graph(G=g, angle=0, colour_edges_by="kind", colour_nodes_by="element_symbol", label_node_ids=False, node_size_min=2, node_alpha=0.85, node_colour_map=plt.cm.tab10)
      4 plt.suptitle("Atom-level graph. Nodes coloured by their Element")

5 frames
[/usr/local/lib/python3.7/dist-packages/matplotlib/artist.py](https://localhost:8080/#) in _update_property(self, k, v)
   1000                 if not callable(func):
   1001                     raise AttributeError('{!r} object has no property {!r}'
-> 1002                                          .format(type(self).__name__, k))
   1003                 return func(v)
   1004 

AttributeError: 'Axes3D' object has no property 'auto_add_to_figure'

![image](https://user-images.githubusercontent.com/51283097/154685890-b7afa3da-4d1d-4d3f-a70c-19f9a8c7d396.png)

**Comment by a-r-j** [comment]
Great!

Similar problem: `!pip install -U matplotlib` will resolve this.

I think the trouble might that the colab runtime already has some versions of these libraries installed, so when you install graphein it doesn't pull in the correct versions. In any case, this is really helpful (thanks!) & I will try to account for this in the dependencies of make it clearer in the colabs.

**Comment by johnnytam100** [comment]
It worked! Thank you!
Btw, there is another cell:

```
# Warning! This cell may crash a binder notebook as the pre-trained model download is rather large!
# NBVAL_SKIP
from graphein.protein.features.sequence.embeddings import esm_sequence_embedding, biovec_sequence_embedding

new_graph_annotation_funcs = {"graph_metadata_functions": [esm_sequence_embedding, biovec_sequence_embedding]}
config = ProteinGraphConfig(**new_graph_annotation_funcs)

g = construct_graph(config=config, pdb_code="3eiy")
print("ESM:", g.graph["esm_embedding_A"])
print("biovec:", g.graph["biovec_embedding_A"])
```

```
DEBUG:graphein.protein.graphs:Deprotonating protein. This removes H atoms from the pdb_df dataframe
DEBUG:graphein.protein.graphs:Detected 174 total nodes
Downloading: "https://github.com/facebookresearch/esm/archive/main.zip" to /root/.cache/torch/hub/main.zip
Downloading: "https://dl.fbaipublicfiles.com/fair-esm/models/esm1b_t33_650M_UR50S.pt" to /root/.cache/torch/hub/checkpoints/esm1b_t33_650M_UR50S.pt
Downloading: "https://dl.fbaipublicfiles.com/fair-esm/regression/esm1b_t33_650M_UR50S-contact-regression.pt" to /root/.cache/torch/hub/checkpoints/esm1b_t33_650M_UR50S-contact-regression.pt
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
[<ipython-input-15-fcdeef6b5567>](https://localhost:8080/#) in <module>()
      6 config = ProteinGraphConfig(**new_graph_annotation_funcs)
      7 
----> 8 g = construct_graph(config=config, pdb_code="3eiy")
      9 print("ESM:", g.graph["esm_embedding_A"])
     10 print("biovec:", g.graph["biovec_embedding_A"])

3 frames
[/usr/local/lib/python3.7/dist-packages/graphein/protein/features/sequence/embeddings.py](https://localhost:8080/#) in _load_biovec_model()
    203     """
    204 
--> 205     return biovec.models.load_protvec(
    206         os.fspath(
    207             Path(__file__).parent.parent

NameError: name 'biovec' is not defined
```

**Comment by a-r-j** [comment]
`pip install biovec`

This should also be installed if you install Graphein with its optional dependencies:

`pip install graphein[extras]`

**Comment by johnnytam100** [comment]
Arian, thanks again! It worked!

**Comment by a-r-j** [comment]
This is now fixed in v1.1.1 (`pip install graphein==1.1.1`). You no longer need to change the versions of `pyyaml` and `matplotlib`. `biovec` still requires additional install (`pip install biovec` / `pip install graphein[extras]==1.1.1`); this is the intended behaviour.

**Comment by johnnytam100** [comment]
Thanks Arian!
